security/Kconfig

```diff
@@ -42,6 +42,21 @@ config SECURITY_TIOCSTI_RESTRICT
 
 	  If you are unsure how to answer this question, answer N.
 
+config SECURITY_SYSFS_RESTRICT
+	bool "Sysfs/debugfs restriction"
```

Comment:
But yon don't handle debugfs as in the last public patch of grsecurity/PaX, why?

Reply:
Debugfs is usually mounted under sysfs so we'd only need to restrict the permissions of sysfs.

Reply:
I believe an explicit handling would be better instead of relying it to be mounted under sysfs

Reply:
Is https://github.com/anthraxx/linux-hardened/pull/22/commits/2d5c76b4b58ffbaae1912c44106ef56d61ec27f8 ok?